aioredis/connection.py

```diff
@@ -858,6 +858,11 @@ async def read_response(self):
         except asyncio.TimeoutError:
             await self.disconnect()
             raise TimeoutError(f"Timeout reading from {self.host}:{self.port}")
+        except OSError as e:
```

Comment:
Seems too overarching. Is there a reason not to catch socket.error and socket.SOMETHING (assuming IIRC redis-py caught two errors from socket) specifically?

Reply:
yes, you are right!

Reply:
https://docs.python.org/3.6/library/socket.html#socket.error

socket.error has been deprecated, socket.error is an alias for `OSError`